printing/printer.ml

```diff
@@ -615,18 +592,14 @@ let print_evar_constraints gl sigma =
         str" with candidates:" ++ fnl () ++ hov 0 ppcandidates
     else mt ()
 
-let should_print_dependent_evars = ref false
-
-let () =
-  let open Goptions in
-  declare_bool_option
-    { optdepr  = false;
-      optkey   = ["Printing";"Dependent";"Evars";"Line"];
-      optread  = (fun () -> !should_print_dependent_evars);
-      optwrite = (fun v -> should_print_dependent_evars := v) }
+let should_print_dependent_evars =
+  Goptions.declare_bool_option_and_ref
+    ~depr:false
+    ~key:["Printing";"Dependent";"Evars";"Line"]
```

Comment:
If you create a wrapper function, it would be a nice touch to give the option as a string ""Printing Dependent Evars Line" and have the wrapper split it.  A bit easier to read.

Reply:
This would require to change the whole `Goptions` interface, because this `string list` type for keys is also used in functions like `Goptions.set_bool_option_value` (see the examples where the key is exported and reused with this function in other places of the code).

Reply:
That function is only called in two places.  Probably not that much change.  But not strictly necessary and entirely your choice.

Reply:
A few more places after this PR.